spy-reaper/spy-reaper.py

```diff
@@ -18,6 +20,13 @@
     except ImportError:
         sys.stderr.write("twsapi 973.05 must be installed under a system path or '{0}'\n".format(alternate_path))
         sys.exit(1)
+        
+#TWS API version check, please see if this works.    
+tws_api_ver = ibapi.get_version_string()
+tws_req_ver = "9.73.2"
+
+if tws_api_ver != tws_req_ver:
+    raise sys.stderr.write("twsapi version '{0}' required, version installed is '{1}'".format(tws_req_ver, tws_api_ver))
```

Comment:
Please follow the error handling convention for system sanity checks:
sys.stderr.write("ERROR MESSAGE\n")
sys.exit(1)

This will terminate the program with a non-zero exit code indicating failure.

Using "raise" without an "except" to catch it will print a full backtrace of exceptions which is hard for the user to understand.

NOTE: We may decide to do error-handling differently elsewhere in the project, but for system sanity checks this is the current convention.